### ISSUE TITLE
Limit the number of conversations sent

### ISSUE THREAD

**Issue by shusain93** [shusain93/OSXMessageProxy#15]
Some people have tons and tons of conversations and it can take way too long to send all of them. Plus, you won't be scrolling back hundreds of conversations, maybe limit 50?

**Comment by Asima13** [comment]
Is there any way I could contact you through PM? I have a few questions for you in regards to this.

**Comment by shusain93** [comment]
Sure I guess? Email me at shusain93@gmail.com

**Comment by shusain93** [comment]
Yeah sure. I added a comment here with my email and then deleted it. If you have email notifications on you should have gotten the message in your inbox.
